### DISCUSSION TITLE
All docs considered in Inbox, no classification model error

### DISCUSSION THREAD

**Discussion by toddmryan** [jonaswinkler/paperless-ng#997] [Support]
These two issues may be related.  Running 1.4.1, 130 documents.  No matter what tags I apply or remove, all documents show up as being in my inbox when I filter by Inbox status (Views-->Inbox).  Even documents with no tags and certainly no inbox tags.  I also still get this in my logs whenever I take any action.

[2021-05-07 18:52:11,063] [DEBUG] [paperless.classifier] Document classification model does not exist (yet), not performing automatic matching.  

I have several dozen tags with rules as well as correspondents and doc types.  But still no classification action.  I'm guessing it might be because it thinks every document still has an inbox tag(?) even though they do not.  I'd be grateful if someone could point me in the right direction on how to get this working.

**Comment by tesmerjg** [comment]
I too am seeing this in the logs: "Document classification model does not exist (yet), not performing automatic matching." How does one establish a classification model?
